### ISSUE TITLE
wakanda server source

### ISSUE THREAD

**Issue by vegarringdal** [Wakanda/wakanda-issues#99]
Is there any plans to update the server source ?

https://github.com/Wakanda/core-Wakanda
https://github.com/Wakanda/core-XToolbox
https://github.com/Wakanda/core-Components

Looks like it havent been updated since wak 9, so guessing you guys have replaced most with v8 etc 😂 
Or isnt the server open source anymore?

I know I wont be able to just pick up the source and help with bug fixes this year (maybe newer) , but looking at something like unreal engine it looks like a lot of the community is helping them catching/patching bugs.

It would prb also be a big comfort for some users/companies knowing that the source is available, and that they have the option to add to/fix a priority bug if its needed with own resources if needed.

**Comment by hamzahik** [comment]
Thanks for asking @vegarringdal 

Being that we share some code with 4D (closed source), putting code automatically online on Github was not possible so it was done manually especially that we were using Perforce and not Git. Until WAK9 code was updated but since we've noticed that almost everyone does not care about the C++ source code but about the product itself and the JavaScript code to be MIT we decided to focus on that. Now JS code is MIT.

For big companies, we do comfort them by giving our code to a specialized organism in France (even 4D closed source code) that ensures big companies who rely heavily on our products that if all 4D/Wakanda employees were to be kidnapped by aliens at the same time(or some more realistic scenario) they can get all the products code and keep maintaining it.

That said we're now using GitLab (instead of Perforce) and this will hopefully make updating the source code more often an easier task. I can't give an ETA though because we are focused on the big 2.x.x evolutions.
